test_apps/smokeTests/gridTools.cpp

```diff
@@ -67,39 +68,60 @@ template<typename T> vector<T *> AllocateBlocksType(const vector<size_t> &bs, co
 
 vector<float *> AllocateBlocks(const vector<size_t> &bs, const vector<size_t> &dims) { return (AllocateBlocksType<float>(bs, dims)); }
 
-void MakeTriangle(Grid *grid, float minVal, float maxVal)
+void MakeTriangle(Grid *grid, float minVal, float maxVal, bool addRandomMissingValues)
 {
     auto   dims = grid->GetDimensions();
     size_t x = dims[X];
     size_t y = dims[Y];
     size_t z = dims[Z];
 
+    std::random_device rd;  //Will be used to obtain a seed for the random number engine
+    std::mt19937 gen(rd()); //Standard mersenne_twister_engine seeded with rd()
+    std::uniform_int_distribution<> distrib(0,1);
+
     float value = minVal;
+    float missingValue = grid->GetMissingValue();
+    int missing=0;
```

Comment:
Is this variable used anywhere? If not, how about not declare it?

Reply:
A diagnostic variable that I missed.  Fixed.

Reply:
Do you think it's gonna be helpful to catch such instances (diagnostic variable that people forgot to delete) automatically by either GCC or clang-tidy? If so, I can open a request.

Reply:
I don't think we need additional checks for this kind of warning, since GCC did catch it in the failed CI test.  I'm sure clang-tidy will notice this as well.  So I don't know what more could be done for compiler warnings.

Reply:
Got it. As long as some automated mechanism is catching it, I'm good :)

Reply:
Yes - the reason this one wasn't obvious is because the smoke tests were already failing.  The warning test didn't really stand out like it normally would :\